src/synthesizer/line.py

```diff
@@ -15,20 +15,133 @@
 
 """
 
-from dataclasses import dataclass, field
-from typing import Dict, List, Tuple
-
 import numpy as np
-from unyt import Angstrom
+from unyt import Angstrom, unyt_array
 
-from synthesizer import exceptions
+from synthesizer import exceptions, line_ratios
 from synthesizer.conversions import lnu_to_llam
 from synthesizer.units import Quantity
 
 
+def vacuum_to_air(wavelength):
```

Comment:
I think these conversion methods should be in the `conversions.py` top level module

Reply:
+1

Reply:
Done